docs/packages/pkg/libscrypt.rst

```diff
@@ -0,0 +1,20 @@
+.. spelling::
+
+    libscrypt
+
+.. index:: crypto ; libscrypt
+
+.. _pkg.libscrypt:
+
+libscrypt
+=========
+
+-  `Official <https://github.com/technion/libscrypt>`__
+-  `Hunterized <https://github.com/hunter-packages/libscrypt>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/libscrypt/CMakeLists.txt>`__
+
+.. code-block:: cmake
```

Comment:
It's better to use `literalinclude` here, see template:
* https://raw.githubusercontent.com/ruslo/hunter/b9ec25a0ffd23eeccdb692edef47f1cd03db30f9/docs/packages/pkg/foo.rst